src/theme/theme.js

```diff
@@ -17,6 +17,9 @@ export const baseTheme = EditorView.theme({
     overflow: 'hidden',
     textOverflow: 'ellipsis'
   },
+  '&.cm-editor': {
```

Comment:
I was wondering whether plain `'.cm-editor'` is enough, but it didn't work when changed 👍